Fineract-1646

apurbraj opened the pull request:
FINERACT-1646 (System is posting multiple Overdraft Interest for same day)

Commit:
Charts data is mandatory for creating a fixed deposit product

Commit:
Fix makerchecker inbox list group by error

renovate-bot committed:
chore(deps): update all non-major dependencies

galovics reviewed (changes requested):
Add a test-case to cover this bug please so it never reoccurs. Thanks.

adamsaghy committed:
Date to LocalDate - part 1

adamsaghy committed:
Date to LocalDate - Client

rrpawar96 committed:
FINERACT-1402:PaymentTypeMandatory

adamsaghy committed:
Fix typo in error messages in Loan

dependabot[bot] committed:
chore(deps): bump actions/setup-node from 3.3.0 to 3.4.0

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 3.3.0 to 3.4.0.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/eeb10cff27034e7acf239c5d29f62154018672fd...5b949b50c3461bbcd5a540b150c368278160234a)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump actions/setup-java from 3.4.0 to 3.4.1

Bumps [actions/setup-java](https://github.com/actions/setup-java) from 3.4.0 to 3.4.1.
- [Release notes](https://github.com/actions/setup-java/releases)
- [Commits](https://github.com/actions/setup-java/compare/16cca5479d7c6b6843f6a6515640ba33c6501543...2c7a4878f5d120bd643426d54ae1209b29cc01a3)

---
updated-dependencies:
- dependency-name: actions/setup-java
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

apurbraj committed:
Fineract-1646

apurbraj committed:
Merge branch 'apache:develop' into develop

apurbrajdhan committed:
Fineract-1646
